In [170]:
import pandas as pd


![title](img/salario.jpg)

# Salarios dos Servidores Brasileiros - Janeiro 2020

In [112]:
df=pd.read_csv('202001_Remuneracao.csv',sep=';',encoding='ISO-8859-1')

In [113]:
#Mudando a configuração de "," para "."
df.iloc[:,5:]=df.iloc[:,5:].apply(lambda x: x.str.replace(',','.'))

In [114]:
#Mudando o dtype para float
df.iloc[:,5:]=df.iloc[:,5:].astype(float)

In [144]:
#Algumas informações relevantes sobre o dataset - já podemos ver algumas informações interessantes..como podemos ter remumeração bruta após deduções negativa?
df.describe()

MES  Id_SERVIDOR_PORTAL  REMUNERAÇÃO BÁSICA BRUTA (R$)  \
count  535862.0        5.358620e+05                  535862.000000   
mean        1.0        9.782748e+06                   10105.817445   
std         0.0        1.958352e+07                    7102.511751   
min         1.0        1.000000e+00                       0.000000   
25%         1.0        2.605960e+06                    4881.292500   
50%         1.0        5.211072e+06                    8037.560000   
75%         1.0        7.817203e+06                   13273.520000   
max         1.0        1.728178e+08                   64795.840000   

       REMUNERAÇÃO BÁSICA BRUTA (U$)  ABATE-TETO (R$)  ABATE-TETO (U$)  \
count                   5.358620e+05    535862.000000         535862.0   
mean                    5.411841e-07        -7.827817              0.0   
std                     3.961608e-04       225.743305              0.0   
min                     0.000000e+00    -25502.500000              0.0   
25%                     0.000000e+00         0.000000              0.0   
50%                     0.000000e+00         0.000000              0.0   
75%                     0.000000e+00         0.000000              0.0   
max                     2.900000e-01         0.000000              0.0   

       GRATIFICAÇÃO NATALINA (R$)  GRATIFICAÇÃO NATALINA (U$)  \
count               535862.000000                    535862.0   
mean                   350.415527                         0.0   
std                   1626.571840                         0.0   
min                  -5214.410000                         0.0   
25%                      0.000000                         0.0   
50%                      0.000000                         0.0   
75%                      0.000000                         0.0   
max                  22157.950000                         0.0   

       ABATE-TETO DA GRATIFICAÇÃO NATALINA (R$)  \
count                             535862.000000   
mean                                  -0.000302   
std                                    0.220812   
min                                 -161.640000   
25%                                    0.000000   
50%                                    0.000000   
75%                                    0.000000   
max                                    0.000000   

       ABATE-TETO DA GRATIFICAÇÃO NATALINA (U$)  ...  \
count                                  535862.0  ...   
mean                                        0.0  ...   
std                                         0.0  ...   
min                                         0.0  ...   
25%                                         0.0  ...   
50%                                         0.0  ...   
75%                                         0.0  ...   
max                                         0.0  ...   

       REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)  \
count                                535862.000000   
mean                                   8787.617462   
std                                    6063.550092   
min                                       0.010000   
25%                                    4541.750000   
50%                                    7195.420000   
75%                                   11715.390000   
max                                   93205.650000   

       REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (U$)  \
count                                 5.358620e+05   
mean                                  4.851995e-07   
std                                   3.551787e-04   
min                                   0.000000e+00   
25%                                   0.000000e+00   
50%                                   0.000000e+00   
75%                                   0.000000e+00   
max                                   2.600000e-01   

       VERBAS INDENIZATÓRIAS REGISTRADAS EM SISTEMAS DE PESSOAL - CIVIL (R$)(*)  \
count                                      535862.000000                          
mean                   

In [143]:
#Vou deletar os salarios negativos para não afetar muito a análise
df=df[df['REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)']>0]


ANO  MES  Id_SERVIDOR_PORTAL             CPF  \
537107  2020  1.0           7400917.0  ***.425.967-**   
537108  2020  1.0            216805.0  ***.651.506-**   
537109  2020  1.0           1307290.0  ***.034.989-**   
537110  2020  1.0           8504240.0  ***.041.463-**   
537111  2020  1.0           8915543.0  ***.857.198-**   

                                NOME  REMUNERAÇÃO BÁSICA BRUTA (R$)  \
537107  ZURLINDE CABRAL LOURO JUNIOR                       11643.75   
537108           ZUY MARIA MAGRIOTIS                       19646.82   
537109          ZUZANA RENDAKI MIKOS                        4801.59   
537110       ZUZITA DOS SANTOS VIANA                        6612.02   
537111                 ZYSMAN NEIMAN                       17945.81   

        REMUNERAÇÃO BÁSICA BRUTA (U$)  ABATE-TETO (R$)  ABATE-TETO (U$)  \
537107                            0.0              0.0              0.0   
537108                            0.0              0.0              0.0   
537109                            0.0              0.0              0.0   
537110                            0.0              0.0              0.0   
537111                            0.0              0.0              0.0   

        GRATIFICAÇÃO NATALINA (R$)  ...  \
537107                         0.0  ...   
537108                         0.0  ...   
537109                         0.0  ...   
537110                         0.0  ...   
537111                         0.0  ...   

        REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)  \
537107                                      9208.71   
537108                                     13624.88   
537109                                      4300.70   
537110                                      5663.08   
537111                                     12370.70   

        REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (U$)  \
537107                                          0.0   
537108                                          0.0   
537109                                          0.0   
537110                                          0.0   
537111                                          0.0   

        VERBAS INDENIZATÓRIAS REGISTRADAS EM SISTEMAS DE PESSOAL - CIVIL (R$)(*)  \
537107                                             595.53                          
537108                                             579.14                          
537109                                             732.18                          
537110                                             588.71                          
537111                                             671.53                          

        VERBAS INDENIZATÓRIAS REGISTRADAS EM SISTEMAS DE PESSOAL - CIVIL (U$)(*)  \
537107                                                0.0                          
537108                                                0.0                          
537109                                                0.0                          
537110                                                0.0                          
537111                                                0.0                          

        VERBAS INDENIZATÓRIAS REGISTRADAS EM SISTEMAS DE PESSOAL - MILITAR (R$)(*)  \
537107                                                0.0                            
537108                                                0.0                            
537109                                                0.0                            
537110                                                0.0                            
537111                                                0.0                            

        VERBAS INDENIZATÓRIAS REGISTRADAS EM SISTEMAS DE PESSOAL - MILITAR (U$)(*)  \
537107                                                0.0                            
537108                                                0.0                            
537109                                                0.0                         

In [152]:
#Começando a análise - qual a média/mediana salarial após deduções dos servidores?
df['REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)'].mean()


8787.617462349854

A média salarial é bem alta se compararmos com o salário mínimo e também maior que o salário de engenheiro em muitos casos...

In [142]:
# Qual o top 10 salários após deduções?
df.sort_values('REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)',ascending=False)[['NOME','REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)']].head(10)


NOME  \
416894                PAULO DO CARMO MARTINS   
342464  MARCIO LUIZ DE FREITAS NAVES DE LIMA   
164395                  FABIO AUGUSTO FUJITA   
390901                  MIRIAM MENNA BARRETO   
445197               RENATO TORRES GONCALVES   
4701                ADRIANA CRISTINA DULLIUS   
152324     EMERSON ABDULMASSIH WOOD DA SILVA   
58820                  ARILDO PINTO DA CUNHA   
220271                 IGOR DALTRO FERNANDES   
146930                 ELIOMAR ANTONIO ALVES   

        REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)  
416894                                     93205.65  
342464                                     69210.46  
164395                                     67988.90  
390901                                     65514.54  
445197                                     62779.48  
4701                                       61110.29  
152324                                     60621.55  
58820                                      60276.24  
220271                                     60151.52  
146930                                     60086.59

 Parece que somente no top 10 salário, TODOS ganharam mais que o teto salarial constitucional de 39 mil R$.

In [159]:
#Quantas pessoas ganharam mais que o teto?
df[df['REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)']>39000][['NOME','REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)']].sort_values('REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)',ascending=False)

NOME  \
416894                PAULO DO CARMO MARTINS   
342464  MARCIO LUIZ DE FREITAS NAVES DE LIMA   
164395                  FABIO AUGUSTO FUJITA   
390901                  MIRIAM MENNA BARRETO   
445197               RENATO TORRES GONCALVES   
...                                      ...   
421794                  PAULO ZARATE PEREIRA   
208309                  GUSTAVO CUNHA GARCIA   
164391   FABIO AUGUSTO DE VASCONCELOS COELHO   
40770             ANDRE LUIS SOARES DE MATOS   
277680                        JULIANA RABELO   

        REMUNERAÇÃO APÓS DEDUÇÕES OBRIGATÓRIAS (R$)  
416894                                     93205.65  
342464                                     69210.46  
164395                                     67988.90  
390901                                     65514.54  
445197                                     62779.48  
...                                             ...  
421794                                     39027.51  
208309                                     39017.25  
164391                                     39015.46  
40770                                      39015.46  
277680                                     39008.23  

[2742 rows x 2 columns]

2742 servidores tiveram uma remuneração após deduções maior que o teto constitucional de 39.000 R$....

In [167]:
#Vamos dar uma olhada nas verbas indezatórias?
df['TOTAL DE VERBAS INDENIZATÓRIAS (R$)(*)'].sum()/1000000

364.95529714

 Só em verbas indenizatórias o governo gastou 365 milhões de reais..

In [169]:
#Vamos dar uma olhada no top 20 servidores que mais ganharam verbas?
df.sort_values('TOTAL DE VERBAS INDENIZATÓRIAS (R$)(*)',ascending=False)[['NOME','TOTAL DE VERBAS INDENIZATÓRIAS (R$)(*)']].head(20)

NOME  \
421985         PEDRO ANTONIO ARRAES PEREIRA   
207859             GUMERCINDO PEREIRA FILHO   
55381        ANTONIO LUIS MACHADO DE MORAES   
348773         MARCOS ROGERIO ALVES RIBEIRO   
223053  IRAILDES MAGALHAES MONTIJO BANDEIRA   
509038                TIMOTHY JOHN BROCKSOM   
518155                  VASSILIOS THEOFILIS   
504887    THEODOMIRO PEREIRA DA SILVA FILHO   
344784                MARCO AURELIO SALTURI   
252990                    JORGE CEZAR COSTA   
3383                   ADILSON ELIAS XAVIER   
258903           JOSE CARLOS GOMES DA SILVA   
38202   ANDERSON JOSE SANT ANNA DE OLIVEIRA   
132826          EDILSON VASCONCELOS QUEIROZ   
440515                    REINALDO TAKARABE   
291714         LAURA TAVARES RIBEIRO SOARES   
293434             LEANDRO FONSECA DA SILVA   
27416         AMELIA MARIA RIBEIRO DE JESUS   
319554         LUIS ROBERTO DA SILVA MACEDO   
10840                        ALAOR CARVALHO   

        TOTAL DE VERBAS INDENIZATÓRIAS (R$)(*)  
421985                               115516.68  
207859                                71508.44  
55381                                 69597.57  
348773                                66887.89  
223053                                63832.77  
509038                                54871.02  
518155                                54746.69  
504887                                54000.49  
344784                                47362.04  
252990                                43203.37  
3383                                  41060.02  
258903                                37761.49  
38202                                 36492.89  
132826                                36492.89  
440515                                36286.53  
291714                                33183.82  
293434                                32100.78  
27416                                 31839.52  
319554                                30614.55  
10840                                 30568.20

# O que será que esses servidores fazem com tanta verba...?